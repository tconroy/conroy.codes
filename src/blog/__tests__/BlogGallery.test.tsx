import {
  render,
} from '@testing-library/react';
import {
  BlogGallery
} from '../BlogGallery';

describe('BlogGallery', () => {
  it('should render the expected results', () => {
    expect(render(
      <BlogGallery
        posts={[]}
        pagination={{}}
      />
    ).container).toMatchSnapshot();
  })
});