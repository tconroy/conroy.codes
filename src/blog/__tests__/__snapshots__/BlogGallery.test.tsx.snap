// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogGallery should render the expected results 1`] = `
<div>
  <ul />
  <div
    class="text-sm flex justify-between"
  />
</div>
`;
